zipkin-lens/.env.production

```diff
@@ -0,0 +1 @@
+BASE_PATH=/zipkin
```

Comment:
So, I think there are two concepts going on
1. what is the base path of the UI (to override in case it isn't "/zipkin"
2. how do you get to the API server (formerly `API_BASE` which is a URI)

Can you separate these out, and then try to match up the README with whatever the outcome is?

Reply:
I separated the two variables API_BASE and BASE_PATH. You are definetely right. It is best to separate the two.

Reply:
So, the README says this:

> By default, API requests are proxied to `http://localhost:9411`. You can change this target using
the `API_BASE` environment variable, e.g., `API_BASE=http://tracing.company.com npm start`.


So being user-empathetic, best to check the README works or change the readme. I think `API_BASE` should be defaulted to `http://localhost:9411` and BASE_PATH should be a path (like /zipkin), not a URI like `BASE_PATH` in the .env.test file.

Can you think through this, and converge with the README or change it and update? If `BASE_PATH` must be a URI not a path, then, we need to note that in the release notes and change the variable to `BASE_URI`

Reply:
The BASE_PATH variable can be anything. I changed it to correctly reflect the values described in the README.